### PR TITLE
Covid Data metrics

### PR DESCRIPTION
**Pull request by sabetech**
In this pull request:
- Consumed the Covid Data API from Narrativa
- Implemented Redux Toolkit for global state management
- Used Router-Dom for routing
- Wrote tests for the components in the project
- Followed the design guideline suggested in the instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,49 @@
...
-### `npm start`
+- **Reactjs** and **Redux** and **Javascript**
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## DEMO
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+[Live Demo Link](https://heroku_link_goes_here.com)
```
Comment (bimbolabuari) [inline README.md]: As required, Kindly deploy your application properly using Heroku, Netlify or GitHub Pages 👍

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @sabetech, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Descriptive Project demo
- Acknowledge [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the author of the original design
- No linter error

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (sabetech) [reply README.md]: Deployed and link updated

**Review by bimbolabuari — APPROVED** [review]
Hi @sabetech,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**